website/sales/templates/sales/order_payment.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'base.html' %}
+{% load i18n payments %}
+
+{% block body %}
+    <section class="page-section">
+        <div class="container">
+            <h1 class="section-title text-center">{% trans "Pay for order" %}</h1>
+            <div class="text-center">
+                {{ order.order_description }}
+            </div>
+            <div class="text-center mt-4">
+                {% url 'index' as next %}
+                {% payment_button order next %}
+                <script type="text/javascript">
```

Comment:
this script should be in a `{% block js_body %}` block (don't forget the super)

Reply:
maybe it would also be smart to give the payments button an id or name so that it is possible to select it by name instead of by index, which may be affected if we somehow add a form to the base template.